rocky/account/forms/account_setup.py

```diff
@@ -168,10 +172,42 @@ class Meta:
         fields = ("account_type", "name", "email", "password")
 
 
-class OrganizationMemberForm(forms.ModelForm):
+class OrganizationMemberEditForm(forms.ModelForm):
+    member_name = forms.CharField(
+        required=False,
+        label=_("Member name"),
+        help_text=_("This name we will use to communicate with you."),
+    )
+    trusted_clearance_level = forms.ChoiceField(
+        required=False,
+        label=_("Trusted clearance level"),
+        choices=SCAN_LEVEL.choices,
+        widget=forms.RadioSelect(),
+        help_text=_("Select a clearance level you trust this member with."),
+    )
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if self.instance.user.is_superuser:
+            self.fields["trusted_clearance_level"].disabled = True
+        if "admin" in self.instance.user.groups.all() or self.instance.user.is_superuser:
+            # There could be a case where you block yourself out of the system
```

Comment:
So it's possible an admin blocks himself? Maybe its better to disable the checkbox for the current user? For sanities sake?

Reply:
> 

Which checkbox? I have disabled the dropdown for admins to change their status.

Reply:
Yes, but an organization admin shouldn't be able to block themselfs. If that's not the case then this is fine.

Reply:
Logic moved to Views, needed the request